File: Arbitration/MPArbitration/ClientApp/src/app/model/payor-group-response.ts
import { Payor } from "./payor";

export class PayorGroupResponse {
    public message = '';
    public itemsAdded = 0;
    public itemsSkipped = 0;
    public itemsUpdated = 0;
    public payorsSkipped:Array<string> = [];

    constructor(obj?:any) {
        if(!obj)
            return;
        this.message = obj.message ?? '';
        this.message = this.message.replaceAll('\n','<br/>');
    }
}

File: Arbitration/MPArbitration/Model/PayorGroupResponse.cs
using System.Text.Json.Serialization;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace MPArbitration.Model
{
    [NotMapped]
    public class PayorGroupResponse
    {
        [JsonPropertyName("itemsAdded")]
        public int ItemsAdded { get; set; } = 0;
        
        [JsonPropertyName("itemsSkipped")]
        public int ItemsSkipped { get; set; } = 0;

        [JsonPropertyName("itemsUpdated")]
        public int ItemsUpdated { get; set; } = 0;

        [JsonPropertyName("payorsSkipped")]
        public string[] PayorsSkipped { get; set; } = { };

        [JsonPropertyName("message")]
        public string Message { get; set; } = string.Empty;
    }
}
